model/build/db.go

```diff
@@ -46,6 +46,8 @@ var (
 	TaskCacheActivatedKey     = bsonutil.MustHaveTag(TaskCache{}, "Activated")
 )
 
+var CompletedStatuses = []string{evergreen.BuildSucceeded, evergreen.BuildFailed}
```

Comment:
put this in the evergreen globals package.

Reply:
:(

Reply:
But that's what we want to write.